plugins/main/public/components/settings/api/api-table.js

```diff
@@ -370,7 +370,7 @@ export const ApiTable = compose(
                           color="primary"
                           iconType="questionInCircle"
                           aria-label="Info about the error"
-                          onClick={() => this.props.copyToClipBoard(item.downReason)}
+                          onClick={() => this.props.copyToClipBoard(api.error.detail)}
```

Comment:
`api` and `api.error` can't become `undefined`, right? Mainly so that there is no reading of `undefined`